Script/olterra.dat

```diff
@@ -210,6 +211,7 @@ decoration
     BitmapPos = 16, 368;
     HPModifier = 25;/*
     IsAlwaysTransparent = false;*/
+    LeftOverItems == banana { Times = 0:1; }
```

Comment:
You can always add banana trees to the olterrains and put a random few in New Attnam